### ISSUE TITLE
Is one of the Generators always supposed to be doing worse than the other?

### ISSUE THREAD

**Issue by jlim13** [junyanz/pytorch-CycleGAN-and-pix2pix#925]
I have G_A and G_B, G_A (Synthetic to Real) seems to consistently have a higher loss than G_B (Real to Synthetic). Is this normal? Any thoughts on how to make G_A's job easier?

**Comment by junyanz** [comment]
It could be. Sometimes the task of A->B might be harder than B->A or vice versa. It's hard to find two domains where both directions are equally hard/easy.

**Comment by jlim13** [comment]
I see, makes sense. Thanks for the response. I'm going to play around with different learning rates for each of the four networks (G_A, G_B, D_A, D_B). I have noticed one of the discriminators going to zero (< 0.09 loss) pretty quickly and staying there.

**Comment by junyanz** [comment]
Sounds great.
